VideoApi/VideoApi.DAL/Queries/GetConferenceHearingRoomsByDateQuery.cs

```diff
@@ -31,14 +31,14 @@ public GetConferenceHearingRoomsByDateQueryHandler(VideoApiDbContext context)
         public Task<List<HearingAudioRoom>> Handle(GetConferenceHearingRoomsByDateQuery query)
         {
 
-            var results = from confStatuses in _context.ConferenceStatuses
-                          join conf in _context.Conferences on confStatuses.ConferenceId equals conf.Id
-                          where conf.AudioRecordingRequired
-                          && confStatuses.ConferenceState == ConferenceState.InSession
-                          && confStatuses.TimeStamp.Date == query.DateStamp.Date
+            var results = from confStatus in _context.ConferenceStatuses
```

Comment:
Could you also change this to conferenceStatus?

Reply:
yes, fixed now. thanks